functions/funtions.py

```diff
@@ -41,23 +40,23 @@ def fit(
     """Train the data
 
     Args:
-        X (numpyarray): train set
-        sim_measure (str, optional): Defaults to "pcc".
-        n_factors (int, optional): Defaults to 100.
-        n_epochs (int, optional): Defaults to 50.
-        learning_rate (float, optional): Defaults to 0.01.
-        algorithm (str, optional): Defaults to "kNN".
+        X (numpyarray): Training data.
+        sim_measure (str, optional):  Similarity measure function. Defaults to "pcc".
+        n_factors (int, optional): number of latent factors. Defaults to 100.
```

Comment:
```suggestion
        n_factors (int, optional): Number of latent factors. Defaults to 100.
```
Either capitalize all first letter of the sentence or not.
It should be consistent across all the docstring.